packages/account/src/Components/poi-idv-submit-complete/__tests__/poi-idv-submit-complete.spec.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import IdvSubmitComplete from '../poi-idv-submit-complete';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
can we have component import last  as it is a relative path. - 
https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules